### PR TITLE
[IPS-1135] Execute the authorization code and refresh token exchange from a backend call

### PR DESCRIPTION
**Pull request by asmclean**
## ✏️ Changes
  
In order to be able to fetch `refresh_token`s, authorization code and refresh token exchanges must be performed from a backend call (rather than from a browser) in order to be able to get `refresh_tokens`.

HAR file demonstrating the new behavior:
[removed]

## 📷 Screenshots
 
No visual changes.
  
## 🔗 References
  
Based on https://github.com/auth0-extensions/auth0-authentication-api-debugger-extension/pull/21.
https://auth0team.atlassian.net/browse/IPS-1135
  
## 🎯 Testing
   
✅ This change has been tested in a Webtask
 
🚫 This change has unit test coverage
  
🚫 This change has integration test coverage
  
🚫 This change has been tested for performance
  
## 🚀 Deployment
  
✅ This can be deployed any time
  
## 🎡 Rollout
  
In order to verify that the deployment was successful we will perform manual testing.
  
## 🔥 Rollback
  
We will rollback if this breaks the extension.
  
### 📄 Procedure
  
Deploy 2.0.0 again.
 
## 🖥 Appliance
  
**Note to reviewers:** ensure that this change is compatible with the Appliance.

### CODE REVIEW

**Comment by asmclean** [comment]
@dctoon, please review.  I've made a few changes from your PR (where I didn't seem to have permission to push).

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.1.0] - 2020-04-03
+
+### Changed
+- Refresh Token Exchanged and Authorization Code exchange will be performed via the backend server.
```
Comment (ziluvatar) [inline CHANGELOG.md:10]:
```suggestion
- Refresh Token Exchange and Authorization Code exchange will be performed via the backend server.
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth0-authentication-api-debugger-extension",
-  "version": "2.0.2",
+  "version": "2.1.0",
```
Comment (dctoon) [inline package.json:3]: @asmclean  shouln't it be a patch version? Now all customers have to explicitly update the version of the extension where with a patch that isn;t needed.

**Comment by CriGoT** [comment]
Snyk alerts are related to dependencies on uglify (used during build) and `auth0@2.17.0` which is the latest version available in all environments

**Pull request merged**